### ISSUE TITLE
Absence of title and search problem in prior knowledge selection

### ISSUE THREAD

**Issue by luizecol** [asreview/asreview#585]
Datafile upload to ASReview contain collumns names used for extract the abstract and title of the article, that can be use in search and random mode. However, in my  case, I had two issue: couldn't use the search mode and, consequently, couldn't see the title in prior knowledge selection because my datafile contain collumns names started in uppercase letter. In this case, ASReview code couldn't recognize the title. 
   As a palliative solution, helped by ASReview people, I download the file I already started screening in .xlsx format, with relevant and irrelevant collumns add by ASReview, change the title collumns "Title" to "title" and "Abstract" to "abstract". After save this file, I upload again to ASReview and it so recognize the title and I could use the search mode, while my previous prior knowledge was preserved. 

In other words, I did the following: 

> start screening without see the title and not able to use search mode
> Use prior knowledge for about 80 articles 
> download the file provided by ASReview model 
> change the collumns name to 'title' and 'abstract' 
> upload again this file to ASReview 
> sees title and able to use search mode 
> my previous prior knowledge was preserved
> keep moving forward 

In this sense, I think it could be useful to change or add some code to recognize collumns that start with uppercase letter.

**Comment by J535D165** [comment]
Thanks for the report. What version of ASReview as you using (`asreview -h`)? I think this issues shouldn't be present in any recent version of ASReview.

**Comment by Rensvandeschoot** [comment]
Dear @luizecol, 

I am closing this issue because [upgrading](https://asreview.readthedocs.io/en/latest/intro/installation.html#upgrade-asreview)to the newest version should solve this issue. Feel free to re-open if it hasn't.
